proto/metrics/agent_payload.proto

```diff
@@ -106,6 +133,11 @@ message SketchPayload {
                 reserved 5, 6;
 				reserved "distributionsK", "distributionsC";
 		repeated Dogsketch dogsketches = 7 [(gogoproto.nullable) = false];
+		// XXX(joey): there exists a metadata field on the SketchPayload as well with a different type. The
```

Comment:
I don't think this comment is relevant/needed in this context

Reply:
Removed